crates/aptos-crypto/src/secp256r1_ecdsa/secp256r1_ecdsa_sigs.rs

```diff
@@ -35,7 +35,7 @@ impl Signature {
 
     /// Deserialize an P256Signature, without checking for malleability
     /// Uses the SEC1 serialization format.
-    #[cfg(not(feature = "fuzzing"))]
+    #[cfg(all(not(feature = "fuzzing"), not(test)))]
```

Comment:
it seems the code for this case the other case are identical.

Reply:
Right, so maybe just leave `pub fn from_bytes_unchecked` without a `#[cfg()]` and remove the duplicated one?

Reply:
yes!